.storybook/main.cjs

```diff
@@ -0,0 +1,34 @@
+module.exports = {
+    core: {
+        builder: 'webpack5',
```

Comment:
Yup it will. 😅  ...that's the way it was done in Fabric. But it was a lot more problems with Storybook (as I mentioned earlier)... this was just an attempt at getting it running at lowest cost. 😅  We decided to save it for later and see what we do with it...